src/rpc.rs

```diff
@@ -1,28 +1,21 @@
 use std::pin::Pin;
 
-use dcs::mission_server::{Mission, MissionServer};
+use dcs::mission_server::Mission;
+use dcs::streamer_server::Streamer;
 use dcs::*;
 use dcs_module_ipc::IPC;
 use futures::{Stream, StreamExt};
-use tonic::transport::server::Router;
-use tonic::transport::{self, Server};
 use tonic::{Request, Response, Status};
 
 pub mod dcs {
     tonic::include_proto!("dcs");
 }
 
 pub struct RPC {
-    ipc: IPC<Event>,
+    pub ipc: IPC<Event>,
```

Comment:
```suggestion
    ipc: IPC<Event>,
```

In case you add the `new` constructor suggested in the next comment

Reply:
Done